main.tf

```diff
@@ -10,6 +10,8 @@ resource "random_id" "id" {
 resource "cloudfoundry_service_instance" "redis" {
   name  = "tf-redis-${local.postfix}"
   space = var.cf_space_id
+  tags                           = var.tags
+  json_params                    = var.json_params
```

Comment:
Does the default (empty string) work ok?

Reply:
I have given it a default value of empty string..

Reply:
I have also fixed the formating issue can you please queue this again..

Reply:
fixed the init issue.. it worked in my local 
Can you please queue again..??